README.md

```diff
@@ -51,7 +51,9 @@ This repository is being development under a task order of the Agile Blanket Pur
   - AWS_REGION
   - AWS_BUCKET_NAME
   - SUDS_API_URL
-    - To point at mocks, update this to be `http://localhost:${PORT}/mocks/basic`
+    - To use the moxai dependency and point at the mock API, update this to be `http://localhost:${PORT}/mocks/basic`
```

Comment:
Should be: `http://localhost:${PORT}/mocks` instead of `http://localhost:${PORT}/mocks/basic`